resources/pvc.py

```diff
@@ -70,21 +70,22 @@ def resize_pvc(self, new_size, verify=False):
         return True
 
 
-def delete_all_pvcs():
+def delete_pvc(pvc_name):
```

Comment:
few things:
1) this function deletes PVCs and not a single PVC so the name should be delete_pvcs()
2) the description is wrong as it deletes multiple PVCs
3) pvc_name parameter should be pvc_names
4) pvc_name (list): List of PVC names for deletion